templates/boundary/abstract.html

```diff
@@ -4,6 +4,7 @@ <h3>Boundary data</h3>
 <p>These data are intended for exposing the polygon boundaries used across SNAP tools and data. Each query for a polygon will return the representative GeoJSON.</p>
 <h4>Service Endpoints</h4>
 <ul>
```

Comment:
I propose simplifying these the templates as part of this PR to reduce the number of files, or, this can be done in a subsequent PR (if you don't want to do it now, make a ticket with this suggested change!):

 1. Move the content from the `area.html` into this file (that's the most important content!!)
 2. Remove the individual templates for each type of place (and remove them from the routing, etc as needed)
 3. Enhance this abstract to list examples for each type of thing,
 4. Add a list of data sources for the boundary polygons (even if it's just our own geospatial vector veracity repo!).

I think simpler is better here.  Less code!

Reply:
Agreed! This is done.